File: u-admin/src/main/java/com/study/u/dataobject/Asset.java
package com.study.u.dataobject;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Data
@Entity
@Table(name = "u_asset")
public class Asset {

    @Id
    private String id;

    private String username;

    /** 投资资产. */
    private Integer invest;

    /** 推广资产.  */
    private Integer extend;

    /** 总资产.  */
    private Integer allUsdt;

}

File: u-android/app/src/main/java/com/study/u/data/Asset.kt
package com.study.u.data

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class Asset(

    var id: String,

    var username: String,

    /** 投资资产. */
    var invest: Int,

    /** 推广资产.  */
    var extend: Int,

    /** 总资产.  */
    var allUsdt: Int

)